src/stateManagement/loadHandlerManager.js

```diff
@@ -33,6 +78,7 @@ const loadHandlerManager = {
   getEndLoadHandler,
   setErrorLoadingHandler,
   getErrorLoadingHandler,
+  removeHandler,
```

Comment:
Could you make this `removeHandlers` (plural)?

Reply:
done

Reply:
It's still exporting the name as `removeHandler`....